controllers/recipeController.js

```diff
@@ -57,6 +62,21 @@ const getSingleRecipe = async (req, res) => {
 */
 const createRecipe = async (req, res) => {
 
+    console.log("inside create recipe")
+
+    // Get user input from the request body
+    const { id } = req.params
+    const { name, description, directions, ingredients, notes, serving } = req.body 
+
+    Recipe.save((err) => {
```

Comment:
Don't forget to create and pass the JSON to the create function. Additionally, the Model object does not have a `save` function, but it does have `create`

Check out the documentation here: https://mongoosejs.com/docs/api/model.html#model_Model-create

Should likely look like `Recipe.create(jsonObject)`
with the json Object being the json representation of the request coming in (the destructed object)

Based on the documentation it looks like `.save() `is middleware, so let's stay away from that and use` .create()`